_pages/ato/controls.md

```diff
@@ -53,6 +61,8 @@ See the [security scanning](../../security/scanning/) page for more details.
 * [SI-2 Flaw Remediation](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=SI-2)
 * [SI-10 Information Input Validation](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=SI-10)
 
+_Note: If you are using 18F's [cloud.gov](https://cloud.gov/) PaaS, the applications will **NOT** inherit the security controls listed above, thus it will be up to your enterprise team to support these controls._
+
 ### Monitoring
 
 One or more of CloudWatch, CloudTrail, New Relic, Splunk, etc.
```

Comment:
It needs to be set up by the project teams, yes. https://github.com/18F/before-you-ship/issues/125